### PR TITLE
Viseo filter

### PR DESCRIPTION
**Pull request by srishtigautam21**
Add:
Video filter and toasts
Files to check:
categoryContext
videofilter.jsx
videCard.jsx
videoListingPage.jsx
toastify.jsx

### CODE REVIEW

**Review by Kavita539 — APPROVED** [review]
- code looks good and clean
- Filter is working properly

File: src/component/VideoFilter/VideoFilter.jsx [hunk 1]
```diff
@@ -0,0 +1,20 @@
+import "./videoFilter.css";
+import { useCategory } from "../../context";
+
+const VideoFilter = ({ categor }) => {
```
Comment (itsmebipul00) [inline src/component/VideoFilter/VideoFilter.jsx:4]: I think categor is also a typo

File: src/context/AuthContext.js [hunk 2]
```diff
@@ -24,13 +25,15 @@ const AuthProvider = ({ children }) => {
...
+    loginLogoutToast("Login successful");
   };
 
   const logOut = () => {
     localStorage.clear();
     setEncodedToken(null);
     setIsUserLoggedIn(false);
     setUser({});
+    loginLogoutToast("Logged out");
   };
```
Comment (itsmebipul00) [inline src/context/AuthContext.js:37]: Appreciation : Nicely done

**Review by zoyatanweer — APPROVED** [review]
Very nice and understandable code. Also the UI looks beautiful. Good job Srishti. Good to merge.

**Review by shashikantmaurya0007 — APPROVED** [review]
Code looks good to be merged

**Comment by itsmebipul00** [comment]
Code is amazing! UI is bright and awsome! Small suggestion is that you could make pages responsive, also video page is not scrollable entirely in my laptop. These could be worked on later. Nice work Srishti! Good to merge.
Screenshot (39)

**Review by itsmebipul00 — APPROVED** [review]

File: src/component/VideoFilter/VideoFilter.jsx (hunk 1 above)
Comment (srishtigautam21) [reply src/component/VideoFilter/VideoFilter.jsx:4]: actually no .....i had a useState named category thats why named it categor

File: src/context/AuthContext.js (hunk 2 above)
Comment (srishtigautam21) [reply src/context/AuthContext.js:37]: thanks :)

**Pull request merged**